### DISCUSSION TITLE
Authentication Delay in Chrome/Edge

### DISCUSSION THREAD

**Discussion by wilczj** [cesanta/mongoose#2290] [Ask questions, get answers] [answered]
Hello,

We're using Mongoose in an application referenced on the Device Dashboard example.  For some unknown reason on Chrome/Edge there's a noticeable login delay of 5-10 seconds.  Firefox seems much quicker.  Just wondering if anyone had any thoughts as to why?

Thanks for any help!

**Comment by scaprile** [comment, marked as answer]
Unknown reasons usually collapse into one known reason once the observer uses a measurement instrument.
The millions of possible unknowns also sensibly reduce when one explains the scenario. I suggest you use Wireshark, and if needed open an issue and explain what you do, what you don't, what you see, what you don't, fill the issue template, etc.

**Comment by wilczj** [comment]
This was a simple just wondering if you’ve seen this before tip of the iceberg question, for the record I do not appreciate the tone of the response.I used my personal GitHub account to ask the question just out of convenience, but this request is from GE, who has a purchased license.

**Reply by scaprile** [reply]
I'm sorry if you did not like something in the response. As always, if you provide data to reproduce it, we'll do it and if it is something related to Mongoose, we'll fix it.
One thing that I may imagine closer to your comment is TLS on a microcontroller with a long cipher, but I have no clue of what you are doing nor with what you are doing it.
Other is that some font data is loaded off a third-party place, in case you are exactly using the same code of the device dashboard.
Otherwise, I can't help with further data like a Wireshark capture, for example

**Comment by wilczj** [comment]
If it’s not a known issue on your end then clearly it’s something on my end.  That’s all I was trying to ask for.

**Comment by cpq** [comment]
@wilczj commercial customers with support option, have a dedicated support line, per signed agreement. Please use that, if you represent a company.

We're always happy to provide assistance, but please make sure your issue/question contains maximum details for us to reproduce - otherwise, it does not even worth the effort trying. There are so many possibilities, as Sergio mentioned.

Please describe your environment, then every step you do, what you expect to see and what you see instead on every step.

**Comment by wilczj** [comment]
Thanks I’ll use the other support option for future discussions if needed.

**Comment by hall666** [comment]
ok
